exercises/isogram/HINT.md

```diff
@@ -0,0 +1,7 @@
+If you find that `testWorksWithGermanLetters` fails even though you think you've implemented it correctly,
+this could be because of the [JVM](https://en.wikipedia.org/wiki/Java_virtual_machine) encoding. Try setting this encoding in the environment variables on
```

Comment:
At this point, I find myself wondering "what does it mean for the JVM to have an encoding set?" Is it possible to succinctly answer that question in this hint, do you think?